projects/hslayers/src/components/styles/styler.service.ts

```diff
@@ -254,11 +259,30 @@ export class HsStylerService {
       } else {
         this.styleObject = {name: 'untitled style', rules: []};
       }
+      this.geostylerWalkaround();
     } catch (ex) {
       this.HsLogService.error(ex.message);
     }
   }
 
+  /**
+   * Tweak geostyler object attributes to mitigate
+   * some discrepancies between opacity and fillOpacity usage
+   */
+  geostylerWalkaround(): void {
```

Comment:
not a mistake but geostyler**Work**around might be better 😉

Reply:
Yeah, seemed suspicious :)